config/canonical.json

```
@@ -7953,8 +7953,11 @@
   },
   "amenity/fast_food|Стардог!s": {
     "count": 58,
+    "countryCodes": ["ru"],
     "tags": {
       "amenity": "fast_food",
+      "brand:wikidata": "Q4439856",
+      "brand:wikipedia": "ru:Стардогс",  
```

Thanks @olgaboiar 👍   
One other thing - when it's an entry in another language like Russian, can you also add `name:en` and `brand:en` tags  (sorry I should have included this in the instructions)

```suggestion
      "brand:wikipedia": "ru:Стардогс",  
      "brand:en": "Stardog!s",
      "name": "Stardog!s",
```